fast-list.js

```diff
@@ -157,12 +162,14 @@ FastList.prototype = {
   },
 
   render: function(reload, changedIndex) {
+    debug('render');
```

Comment:
since we already have the `debugViewport` maybe we don't need to commit this debug